src/applications/ezr/config/chapters/householdInformation/dependentEducationExpenses.js

```diff
@@ -8,6 +8,7 @@ import { replaceStrValues } from '../../../utils/helpers/general';
 import { validateCurrency } from '../../../utils/validation';
 import { LAST_YEAR } from '../../../utils/constants';
 import content from '../../../locales/en/content.json';
+import { DependentEducationExpensesDescription } from '../../../components/FormDescriptions';
```

Comment:
Update this import upon completing the adjustment for the file naming and export convention above.